crowbar_framework/config/repos-cloud.yml

```diff
@@ -97,3 +97,75 @@ suse-12.1:
       required: "recommended"
       url:
       ask_on_error: false
+opensuse-42.1:
+  x86_64:
+    leap-42_1:
+      name: "openSUSE-Leap"
+      required: "mandatory"
+      features: ["os", "ha", "ceph"]
+      repomd:
+        tag: ["obsproduct://build.opensuse.org/openSUSE:Leap:42.1/openSUSE/42.1/ftp/x86_64",
+              "obsproduct://build.opensuse.org/openSUSE:Leap:42.1/openSUSE/42.1/cd/x86_64"]
+        key_md5: "d8039df72c407bc7dd7c1d9c66d58627"
+      url: "http://download.opensuse.org/distribution/leap/42.1/repo/oss/"
+      smt_path:
+      ask_on_error: false
+    leap-42_1-update:
+      name: "openSUSE-Leap-Update"
+      required: "mandatory"
+      features: ["os", "ha", "ceph"]
+      repomd:
+        tag: "obsrepository://build.opensuse.org/openSUSE:Leap:42.1:Update/standard"
+        key_md5: "d8039df72c407bc7dd7c1d9c66d58627"
+      url: "http://download.opensuse.org/update/leap/42.1/oss/"
+      smt_path:
+      ask_on_error: false
+    chef-10:
+      name: "Chef-10"
+      required: "mandatory"
+      #repomd:
+      #  tag: "obsrepository://build.opensuse.org/systemsmanagement:chef:10/openSUSE_Leap_42.1"
+      #  key_md5: "52a2a3f69993789a5362844ec81442dd"
+      #url: "http://download.opensuse.org/repositories/systemsmanagement:/chef:/10/openSUSE_Leap_42.1"
+      repomd:
+        tag: "obsrepository://build.opensuse.org/home:vuntz:crowbar:3.0/openSUSE_Leap_42.1"
+        key_md5: "aef42abc9d4311b5f72f46db225f2575"
+      url: "http://download.opensuse.org/repositories/home:/vuntz:/crowbar:/3.0/openSUSE_Leap_42.1"
```

Comment:
But the crowbar repo got a higher build priority and you should be a maintainer of it :)